Sources/Sentry/include/NSURLSessionTask+Sentry.m

```diff
@@ -0,0 +1,24 @@
+#import "NSURLSessionTask+Sentry.h"
+
+
+@implementation SentryNSURLSessionTask
+
++ (nullable NSString *)graphQLOperationNameFromTask:(NSURLSessionTask *_Nonnull)task
```

Comment:
`m`: It would be great to have some unit tests to cover all the edge cases in this class.